src/protocol/xproto.rs

```diff
@@ -14263,7 +14263,7 @@ impl<'input> QueryTextExtentsRequest<'input> {
     /// Parse this request given its header, its body, and any fds that go along with it
     pub fn try_parse_request(header: RequestHeader, value: &'input [u8]) -> Result<Self, ParseError> {
         validate_request_pieces(header, value, Some(QUERY_TEXT_EXTENTS_REQUEST), None)?;
-        // TODO: Parse odd_length
+        let remaining = &[];
```

Comment:
This change to the generated code seems odd. It should cause a compiler warning about an unused `remaining` variable.

Reply:
It does, which is fixed by a later commit in the series.